docs/streams/upgrade-guide.html

```diff
@@ -213,6 +213,11 @@ <h4>Early Access of the Streams Rebalance Protocol</h4>
         Set <code>unstable.feature.versions.enable=true</code> for controllers and brokers, and
         set <code>unstable.api.versions.enable=true</code> on the brokers as well. In your Kafka Streams application
         configuration, set <code>group.protocol=streams</code>.
+        Besides, the `streams.version` also needs to be upgraded. To do this, run
```

Comment:
I suggest "Upgrade the `streams.version` feature to 1" rather than the sentence which begins "besides".